02 Sample App/00 Boilerplate/package.json

```diff
@@ -43,10 +43,10 @@
     "bootstrap": "^3.3.7",
     "core-js": "^2.4.1",
     "jquery": "^3.1.1",
-    "lc-form-validation": "^0.1.7",
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "react-router": "^3.0.2",
-    "toastr": "^2.1.2"
+    "toastr": "^2.1.2",
+    "validate.js": "^0.11.1"
```

Comment:
If it's just edit training and no validation should we remove validate.js? and add the step later on?